src/rev_ai/apiclient.py

```diff
@@ -40,7 +41,7 @@ def __init__(self, access_token):
         self.session = requests.Session()
         self.session.headers.update({
             'Authorization': 'Bearer {}'.format(access_token),
-            'User-Agent': 'python_sdk'
+            'User-Agent': 'python_sdk-{}'.format(__version__)
```

Comment:
are f strings available for us here?

Reply:
No, f strings were introduced in python 3.6 to my knowledge.

Reply:
i was afraid so :(